server/service/osquery.go

```diff
@@ -818,14 +818,31 @@ func (svc *Service) SubmitDistributedQueryResults(
 				ll = level.Error(svc.logger)
 			}
 			ll.Log("query", query, "message", messages[query], "hostID", host.ID)
-
 		}
+
+		// live queries we do want to ingest even if the query had issues, because we want to inform the user of these
+		// issues
+		// same applies to policies, since it's a 3 state result, one of them being failure
```

Comment:
Mhm... to keep functionality the same we should do the same for labels, see https://github.com/fleetdm/fleet/blob/e3a626a00d4333bdcea2425edfe6f52078952433/server/datastore/mysql/labels.go#L360-L370.

Reply:
Hm... yeah, I suppose you're right. On it...